extract_information.py

```diff
@@ -1,9 +1,9 @@
 from pymongo import MongoClient
-from langchain.embeddings.openai import OpenAIEmbeddings
-from langchain.vectorstores import MongoDBAtlasVectorSearch
-from langchain.document_loaders import DirectoryLoader
-from langchain.llms import OpenAI
 from langchain.chains import RetrievalQA
```

Comment:
i am having issue in this line as i think it is also deprecated
this is also causing issue in from_chain_type as it says no module as this 
Can you help with this?